react/hooks/useScroll.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+
+/**
+ * Hook that handles the scroll position
+ * @returns {Number} - scroll position value
+ */
+const useScroll = () => {
+  const [scroll, setScroll] = useState(0)
+
+  useEffect(() => {
+    const handleScroll = () => {
+      setScroll(window.scrollY)
```

Comment:
this could be throttled, for better performance

Reply:
Done ✅